docs/ops/deployment/kubernetes.md

```diff
@@ -145,18 +204,35 @@ spec:
       containers:
       - name: taskmanager
         image: flink:latest
-        args:
-        - taskmanager
+        workingDir: /opt/flink
+        command: ["/bin/bash", "-c", "$FLINK_HOME/bin/taskmanager.sh start; \
+          while :;
+          do
+            if [[ -f $(find log -name '*taskmanager*.log' -print -quit) ]];
+              then tail -f -n +1 log/*taskmanager*.log;
+            fi;
+          done"]
```

Comment:
Same here, by adding it to the `entrypoint.sh` one could remove a bit of code duplication.

Reply:
Since I already added `command` in deployment, the default `entrypoint.sh` [would be ignored](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes).